langchain/document_loaders/telegram.py

```diff
@@ -217,24 +218,24 @@ def _combine_message_texts(
 
     def load(self) -> List[Document]:
         """Load documents."""
-        if self.chat_url is not None:
+
+        if self.chat_entity is not None:
             try:
                 import nest_asyncio
-                import pandas as pd
 
                 nest_asyncio.apply()
                 asyncio.run(self.fetch_data_from_telegram())
             except ImportError:
-                raise ValueError(
-                    "please install with `pip install nest_asyncio`,\
-                                 `pip install nest_asyncio` "
-                )
```

Comment:
could me make these ImportErrors and error message "x package not found, please install with..."

Reply:
Yeah, I agree.

Personally I prefer adding extras, but here after adding telegram extras, it had to trigger update of all deps in the lock file, so I opted-out from doing so.

Reply:
fixed.